src/test/kotlin/fi/metatavu/timebank/api/test/functional/data/TestTimeEntriesData.kt

```diff
@@ -23,178 +23,181 @@ class TestTimeEntriesData {
-                createTestTimeEntry(
-                    id = 11,
-                    person = 2,
-                    task = 123,
-                    nonProjectTime = null,
-                    timeRegistered = 400,
-                    date = getThirtyDaysAgoSecondWeek()[4].toString(),
-                    createdAt = getODT(getThirtyDaysAgoSecondWeek()[4].atStartOfDay()),
-                    updatedAt = getODT(getThirtyDaysAgoSecondWeek()[4].atStartOfDay())
-                ),
-                createTestTimeEntry(
-                    id = 12,
-                    person = 1,
-                    task = 123,
-                    nonProjectTime = null,
-                    timeRegistered = 312,
-                    date = getThirtyDaysAgoSecondWeek()[5].toString(),
-                    createdAt = getODT(getThirtyDaysAgoSecondWeek()[5].atStartOfDay()),
-                    updatedAt = getODT(getThirtyDaysAgoSecondWeek()[5].atStartOfDay())
-                ),
-                createTestTimeEntry(
-                    id = 13,
-                    person = 2,
-                    task = null,
-                    nonProjectTime = 228255,
-                    timeRegistered = 213,
-                    date = getThirtyDaysAgoSecondWeek()[6].toString(),
-                    createdAt = getODT(getThirtyDaysAgoSecondWeek()[6].atStartOfDay()),
-                    updatedAt = getODT(getThirtyDaysAgoSecondWeek()[6].atStartOfDay())
-                ),
-                createTestTimeEntry(
-                    id = 14,
-                    person = 5,
-                    task = 123,
-                    nonProjectTime = null,
-                    timeRegistered = 435,
-                    date = getThirtyDaysAgoThirdWeek().first().toString(),
-                    createdAt = getODT(getThirtyDaysAgoThirdWeek().first().atStartOfDay()),
-                    updatedAt = getODT(getThirtyDaysAgoThirdWeek().first().atStartOfDay())
-                ),
-                createTestTimeEntry(
-                    id = 15,
-                    person = 1,
-                    task = 123,
-                    nonProjectTime = 228255,
-                    timeRegistered = 435,
-                    date = getThirtyDaysAgoThirdWeek()[1].toString(),
-                    createdAt = getODT(getThirtyDaysAgoThirdWeek()[1].atStartOfDay()),
-                    updatedAt = getODT(getThirtyDaysAgoThirdWeek()[1].atStartOfDay())
-                ),
-                createTestTimeEntry(
-                    id = 16,
-                    person = 1,
-                    task = 123,
-                    nonProjectTime = null,
-                    timeRegistered = 42,
-                    date = getSixtyDaysAgo().toString(),
-                    createdAt = getODT(getSixtyDaysAgo().atStartOfDay()),
-                    updatedAt = getODT(getSixtyDaysAgo().atStartOfDay())
-                )
+            val entries = mutableListOf(
+                    createTestTimeEntry(
```

Comment:
The only changes in this file are the double indentations, this should be undone.

Reply:
What Josh said.
Is the indentation of your IntelliJ IDEA configured to 4 or 2 tabs/spaces? That can be found on the bottom right corner of the IDE.

Reply:
It is 4 spaces for me.